tests/e2e/release-e2e-scenarios.md

```
@@ -636,3 +668,200 @@ sleep 6
 curl -sS "$BASE_URL/admin/api/v1/audit/log?request_id=$REQUEST_ID&limit=5" \
   | jq '{total,entries:(.entries|map({request_id,path,model,provider,status_code,error_type}))}'
 ```
+
+## 12. Authenticated runtime features
+
+### S63 Auth-enabled dashboard runtime config
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)